From c3cfb3213c9ac33679e4b9f01379e6520b769155 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 28 Aug 2013 15:42:34 +0200
Subject: [PATCH] Revert "cgroup.c: check return value of
 unit_realize_cgroup_now()"

This reverts commit 092ce0fbd755a56fe2fd25a28519fe974ebf1d2f.
---
 src/core/cgroup.c | 9 ++-------
 1 file changed, 2 insertions(+), 7 deletions(-)

diff --git a/src/core/cgroup.c b/src/core/cgroup.c
index 50b17f3..5a1c3ad 100644
--- a/src/core/cgroup.c
+++ b/src/core/cgroup.c
@@ -432,13 +432,8 @@ static int unit_realize_cgroup_now(Unit *u) {
                 return 0;
 
         /* First, realize parents */
-        if (UNIT_ISSET(u->slice)) {
-                int r;
-
-                r = unit_realize_cgroup_now(UNIT_DEREF(u->slice));
-                if (r < 0)
-                        return r;
-        }
+        if (UNIT_ISSET(u->slice))
+                unit_realize_cgroup_now(UNIT_DEREF(u->slice));
 
         /* And then do the real work */
         return unit_create_cgroups(u, mask);
